ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/views/ManageSubscriptionsView.kt

```diff
@@ -45,82 +48,77 @@ internal fun ManageSubscriptionsView(
     screen: CustomerCenterConfigData.Screen,
     localization: CustomerCenterConfigData.Localization,
     support: CustomerCenterConfigData.Support,
+    appearance: CustomerCenterConfigData.Appearance,
     modifier: Modifier = Modifier,
     purchaseInformation: PurchaseInformation? = null,
     onAction: (CustomerCenterAction) -> Unit,
 ) {
     Box(
-        modifier = modifier
-            .fillMaxSize()
-            .padding(horizontal = 16.dp),
+        modifier = modifier.fillMaxSize(),
         contentAlignment = Alignment.TopCenter,
     ) {
         Column(
```

Comment:
Hmm do we need the outer `Box`? I think we could remove it and just add the `fillMaxSize` here?